NETCORE/test/Microsoft.ApplicationInsights.AspNetCore.Tests/TelemetryInitializers/ClientIpHeaderTelemetryInitializerTests.cs

```diff
@@ -90,43 +89,36 @@ public void InitializeSetsIpFromRemoteIpAddress()
             Assert.Equal("1.2.3.4", requestTelemetry.Context.Location.Ip);
         }
 
-        [Fact]
-        public void InitializeSetsIPFromStandardHeader()
+        [Theory]
+        [InlineData("X-Forwarded-For", "127.0.0.3", null, "127.0.0.3")]
+        [InlineData("X-Forwarded-For", "127.0.0.3:80", null, "127.0.0.3")]
+        [InlineData("X-Forwarded-For", "[::1]:80", null, "::1")]
+        [InlineData("HEADER", "127.0.0.3;127.0.0.4", ",;", "127.0.0.3")]
+        [InlineData("X-Forwarded-For", "bad", null, null)]
```

Comment:
Can you include some test cases for IPv6?

Reply:
I can add more, `[::1]:80` here was the new ipv6 case I added

Reply:
oh i see. 
sorry, i'm not familiar with ipv6 addresses.

apparently there's two formats:
- short form: `::1`
- long form: `0:0:0:0:0:0:0:1`

i would expect to see unit tests to cover both cases. :)